lib/assert.js

```diff
@@ -103,8 +106,7 @@ function fail(actual, expected, message, operator, stackStartFn) {
         'DEP0094'
       );
     }
-    if (argsLen === 2)
-      operator = '!=';
+    if (argsLen === 2) operator = '!=';
```

Comment:
This change appears to be unrelated.

Reply:
Prettier stuff ^^ You can erase the change if you want :)